travis/travis_install_nightly

```diff
@@ -32,6 +32,11 @@ ln -s `which nodejs` $HOME/maintainer-quality-tools/travis/node
 npm install -g less less-plugin-clean-css
 
 
+# Update PhantomJS (v10 compat)
+npm install --prefix ${TRAVIS_BUILD_DIR} phantomjs-prebuilt@2.1.14
```

Comment:
Using a `PHANTOMJS_VERSION` env var seems good, yeah. 

I would prefer to default upgrade though, instead of defaulting to the old version. As @Yajo mentioned, we have been held back by the old version artificially before it outright broke us.